src/rules/noUnusedVariableRule.ts

```diff
@@ -65,7 +65,14 @@ export class Rule extends Lint.Rules.AbstractRule {
     };
     /* tslint:enable:object-literal-sort-keys */
 
-    public static FAILURE_STRING_FACTORY = (name: string) => `Unused variable: '${name}'`;
+    public static FAILURE_TYPE_BIND_ELEM = "binding element";
```

Comment:
Not so sure about this one -- it might be confusing for beginners, so I'm inclined to continue referencing these values as simply "variables".